src/org/zaproxy/zap/extension/pscanrulesBeta/resources/Messages.properties

```diff
@@ -3,8 +3,8 @@
 pscanbeta.desc = Passive Scan Rules - beta
 
 pscanbeta.authenticationcredentialscaptured.name=Authentication Credentials Captured
-pscanbeta.authenticationcredentialscaptured.desc=An insecure authentication mechanism is in use.  This allows an attacker on the network access to the userid and password of the authenticated user. For Basic Authentication, the attacker must merely monitor the network traffic until a Basic Authentication request is received, and then base64 decode the username and password.  For Digest Authentication, the attacker has access to the username, and possibly also the password, if the hash (including a nonce) can be successfully cracked, or if a Man-In-The-Middle attack is mounted.     
-pscanbeta.authenticationcredentialscaptured.soln=Use HTTPS, and use a secure authentication mechanism that does not transmit the userid or password in an un-encrypted fashion.  In particular, avoid use of the Basic Authentication mechanism, since this trivial obfuscation mechanism is easily broken. 
```

Comment:
Formatting change should be separate from code changes:
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines